spec/features/user_adds_a_new_piece_of_music_spec.rb

```diff
@@ -22,9 +22,11 @@
       visit new_piece_path
       fill_in 'Title', with: "Marriage of Figaro"
       fill_in 'Composer', with: "Mozart"
+      fill_in 'YouTube URL', with: "https://www.youtube.com/watch?v=0FF4HyB77hQ"
       click_button "Add Piece"
 
       expect(page).to have_content "Piece added successfully"
+      expect(page).to have_css('iframe')
```

Comment:
expect(page).to have_css("iframe[url='https://www.youtube.com/embed/0FF4HyB77hQ']")